### ISSUE TITLE
bed files minimum track height thresholded at 40 pixels

### ISSUE THREAD

**Issue by malcook** [igvteam/igv#810]
In my hands, display of .bed files disallows setting the track height below 40 either interactively or using, e.g., maxHeightPixels=15:15:15 in the trackline.

**Comment by jrobinso** [comment]
Yes there are minimum heights for each track type that should be documented somewhere.

**Comment by malcook** [comment]
ok - but I can't find such documentation.  The only mention of max/min heights I can find is:

>  Select Tracks>Fit to Window.
> IGV displays all tracks. If necessary, it sets the track height to 1 pixel and scrolls the data.

Regardless... can you consider dialing down the minimum to 10 or 15 across-the board.  I use 15 often to good effect on other track types and would like the option for .bed files as well.

**Comment by jrobinso** [comment]
Yes but its not as simple as just changing a constant.  The Bed track renders features at a constant height and needs 40 pixels for feature + name.   So the rendering needs changed to adjust to height to support 10 or 15.
